### DISCUSSION TITLE
An idea about gameplay of traitor mode

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#12245] [Ideas and suggestions]
Add point shop for traitor mode
Crews can purchase various things in the point shop
Traitors have access to unique point shops, such as:
1. Add a back door to a weapon or tool. When they trigger, the back door will detonate and injure the user.
2. A lock picker that can quickly open the door
3. Signal jammer, which can be attached to the wall, its sprite depth > 0.800, after loading the battery and turning on the power, it can make the headset within 5 meters unable to send or receive signals
4. Fake handcuffs, players who are handcuffed by the fake handcuffs can remove the handcuffs by themselves
5. Loosen the vent
6. Spitball (Head Slot), making people unable to speak
7. Hacker terminal, you can use other people's steam id to send messages, and can specify who can't receive the message

**Comment by Ek-Videogames** [comment]
Overhauled "only visible to traitors" crafting recipes in the fabricator as well - not only the ones granted by a specific mission.
